playground/AzureSearchEndToEnd/AzureSearch.ApiService/Program.cs

```diff
@@ -12,7 +12,7 @@
 var builder = WebApplication.CreateBuilder(args);
 
 builder.AddServiceDefaults();
-builder.AddAzureSearch("search");
+builder.AddSearchIndexClient("search");
```

Comment:
I think we'd want `Azure` on all these still right?

Reply:
Maybe, https://github.com/dotnet/aspire/issues/1510#issuecomment-1965571299

Do we need a meeting or wait for more opinions here?

Reply:
Let's get some opinions here.

Reply:
Agree Azure in the names